utils/responseHandler.go

```
@@ -18,6 +24,11 @@ func (Success *successPackage) NewDiscordResponse(response http.ResponseWriter,
 			return
 		}
 	} else {
-		response.Write([]byte(`{"success": true, "status": 200, "message": "` + message + `"}`))
+		res, _:= Json.ToJson(SuccessResponse{
+			Success: true,
+			Message: message,
+			Status:  200,
+		})
+		response.Write([]byte(res))
```

golangci-lint (1.64.8)

Error return value of `response.Write` is not checked

(errcheck)